app/services/analytics_events.rb

```diff
@@ -976,6 +976,7 @@ def idv_doc_auth_ssn_visited(**extra)
   # @param [String] back_image_fingerprint Fingerprint of back image data
   # @param [String] getting_started_ab_test_bucket Which initial IdV screen the user saw
   # @param [String] phone_question_ab_test_bucket Prompt user with phone question before doc auth
+  # @param [String] phone_with_camera the result of the phone question a/b test
```

Comment:
also missed in the original PR, but this is a boolean right?
```suggestion
  # @param [Boolean] phone_with_camera the result of the phone question a/b test
```